docs/zh_CN/Release.rst

```diff
@@ -0,0 +1,1120 @@
+
+
+* 修复 ``nnictl update`` 不一致的命令行风格
+* SMAC Tuner 支持导入数据
+* 支持 Experiment 状态从 ERROR 回到 RUNNING
+* 修复表格的 Bug
+* 优化嵌套搜索空间
+* 优化 'randint' 类型，并支持下限
+* `超参调优算法的对比 <CommunitySharings/HpoComparison.rst>`__
+* `NAS 算法对比 <CommunitySharings/NasComparison.rst>`__
+* `NNI 在推荐上的应用 <CommunitySharings/RecommendersSvd.rst>`__
+
+发布 0.7 - 4/29/2018
+-----------------------
+
+主要功能
+^^^^^^^^^^^^^^
+
+
+* `在 WIndows 上支持 NNI <Tutorial/InstallationWin.rst>`__
+
+  * NNI 可在 Windows 上使用本机模式
+
+* `全新 advisor: BOHB <Tuner/BohbAdvisor.rst>`__
+
+  * 支持新的 BOHB Advisor，这是一个健壮而有效的超参调优算法，囊括了贝叶斯优化和 Hyperband 的优点
+
+* `支持通过 nnictl 来导入导出 Experiment 数据 <Tutorial/Nnictl.rst>`__
+
+  * 在 Experiment 执行完后，可生成分析结果报告
+  * 支持将先前的调优数据导入到 Tuner 和 Advisor 中
+
+* `为 NNI Trial 任务指定 GPU 设备 <Tutorial/ExperimentConfig.rst#localConfig>`__
+
+  * 通过 gpuIndices 配置来为 Trial 任务指定GPU。如果 Experiment 配置文件中有 gpuIndices，则只有指定的 GPU 会被用于 NNI 的 Trial 任务。
+
+* 改进 Web 界面
+
+  * 在 Web 界面上使用十进制格式的指标
+  * 添加多阶段训练相关的提示
+  * 可将超参复制为 Python dict 格式
+  * 可将提前终止的 Trial 数据传入 Tuner。
+
+* 为 nnictl 提供更友好的错误消息
+
+  * 为 YAML 文件格式错误提供更有意义的错误信息
+
+Bug 修复
+^^^^^^^^^^^^^^
+
+
+* 运行 nnictl stop 的异步 Dispatcher 模式时，无法杀掉所有的 Python 线程
+* nnictl --version 不能在 make dev-install 下使用
+* OpenPAI 平台下所有的 Trial 任务状态都是 'WAITING'
+
+发布 0.6 - 4/2/2019
+----------------------
+
+主要功能
+^^^^^^^^^^^^^^
+
+
+* `版本检查 <TrainingService/PaiMode.rst>`__
+
+  * 检查 nniManager 和 trialKeeper 的版本是否一致
+
+* `为早停的任务报告最终指标 <https://github.com/microsoft/nni/issues/776>`__
+
+  * 如果 includeIntermediateResults 为 true，最后一个 Assessor 的中间结果会被发送给 Tuner 作为最终结果。 includeIntermediateResults 的默认值为 false。
+
+* `独立的 Tuner/Assessor <https://github.com/microsoft/nni/issues/841>`__
+
+  * 增加两个管道来分离 Tuner 和 Assessor 的消息
+
+* 使日志集合功能可配置
+* 为所有 Trial 增加中间结果的视图
+
+Bug 修复
+^^^^^^^^^^^^^^
+
+
+* `为 OpenPAI 增加 shmMB 配置 <https://github.com/microsoft/nni/issues/842>`__
+* 修复在指标为 dict 时，无法显示任何结果的 Bug。
+* 修复 hyperband 中浮点类型的计算问题
+* 修复 SMAC Tuner 中搜索空间转换的错误
+* 修复 Web 界面中解析 Experiment 的错误格式
+* 修复 Metis Tuner 冷启动时的错误
+
+发布 0.5.2 - 3/4/2019
+------------------------
+
+改进
+^^^^^^^^^^^^
+
+
+* 提升 Curve fitting Assessor 的性能。
+
+文档
+^^^^^^^^^^^^^
+
+
+* 发布中文文档网站：https://nni.readthedocs.io/zh/latest/
+* 调试和维护：https://nni.readthedocs.io/zh/latest/Tutorial/HowToDebug.html
+* Tuner、Assessor 参考：https://nni.readthedocs.io/zh/latest/sdk_reference.html#tuner
+
+Bug 修复和其它更新
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+* 修复了在某些极端条件下，不能正确存储任务的取消状态。
+* 修复在使用 SMAC Tuner 时，解析搜索空间的错误。
+* 修复 CIFAR-10 样例中的 broken pipe 问题。
+* 为本地训练服务和 NNI 管理器添加单元测试。
+* 为远程服务器、OpenPAI 和 Kubeflow 训练平台在 Azure 中增加集成测试。
+* 在 OpenPAI 客户端中支持 Pylon 路径。
+
+发布 0.5.1 - 1/31/2018
+-------------------------
+
+改进
+^^^^^^^^^^^^
+
+
+* 支持配置 `log 目录 <https://github.com/microsoft/nni/blob/v0.5.1/docs/ExperimentConfig.md>`__ 
+* 支持 `不同级别的日志 <https://github.com/microsoft/nni/blob/v0.5.1/docs/ExperimentConfig.md>`__，使其更易于调试。
+
+文档
+^^^^^^^^^^^^^
+
+
+* 重新组织文档，新的主页位置：https://nni.readthedocs.io/en/latest/
```

Comment:
use `https://nni.readthedocs.io/zh/latest/`

Reply:
updated it.